angular.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "metrics.prx.org": {
+      "root": "",
+      "sourceRoot": "src",
+      "projectType": "application",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist",
+            "index": "src/index.html",
+            "main": "src/main.ts",
+            "tsConfig": "src/tsconfig.app.json",
+            "polyfills": "src/polyfills.ts",
+            "assets": [
+              "src/assets",
+              {
+                "glob": "*",
+                "input": "node_modules/ngx-prx-styleguide/assets/images",
+                "output": "assets/images/"
+              }
+            ],
+            "styles": [
+              "node_modules/c3/c3.css",
+              "node_modules/pikaday/css/pikaday.css",
+              "node_modules/pikaday/css/triangle.css",
+              "node_modules/ngx-prx-styleguide/assets/styles/bundle.scss",
```

Comment:
You'll need the `ng-select` stylesheet here if you ever wanted to use that element (or the new tagging element Rick's building) from Styleguide.

Reply:
long story, that multiselect got moved out of metrics a while ago on purpose and it has its own dropdown/select menus, so _if_ it gets added back in, then I'll add the stylesheet.